src/App.js

```diff
@@ -60,19 +67,63 @@ class App extends Component {
   }
 
   resetAllGames = (type) => {
-    const games = this.state.games;
-    let filteredGames;
+
+    let filterOptions = {
+      type: {
+        card: false,
+        board: false
+      },
+      players: {
+        two: false,
+        threefour: false,
+        fivesix: false,
+        seven: false
+      },
+      age: {
+        lesseight: false,
+        eightthirteen: false,
+        thirteenplus: false,
+        adult: false
+      },
+      genre: {
+        strategy: false,
+        family: false,
+        party: false,
+        adventure: false
+      }
+    }
```

Comment:
There should be a more elegant way of getting this "reset" filterOptions state object, but was using it to get it working

Reply:
Regardless, I think it's cool the new look of our reset function, I can clearly and quickly see what it does! Thanks @foxwellm !